### DISCUSSION TITLE
Pinning on W10 taskbar not working correctly

### DISCUSSION THREAD

**Discussion by Sophist-UK** [valinet/ExplorerPatcher#1478] [General]
I can pin running apps to the taskbar, and close them, but when I open them again rather than the pin icon becoming a running task icon, instead I get a separate taskbar entries for the pinned app and the running app.

**Comment by pyrates999** [comment]
Please take a screen shot of what you mean and upload it here as a jpg.

**Comment by Sophist-UK** [comment]
I pin Outlook to the taskbar, and click it to start it and this is what I get:
![image](https://github.com/valinet/ExplorerPatcher/assets/3001893/f31efb31-4f21-41d9-9208-4e5748261838)

**Comment by pyrates999** [comment]
Did you uninstall explorer patcher and reboot to confirm if this is caused by explorer patcher?

**Comment by Sophist-UK** [comment]
I switched back to Windows 11 taskbar (using Properties) and it was fine - and then back to Windows 10 taskbar and had the same issue.

I cannot see how uninstalling Explorer Patcher and rebooting would do anything different as uninstalling will switch back to Win 11 taskbar as above.

**Comment by pyrates999** [comment]
Good, then you've confirmed it.  Thank you.
